src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -5012,7 +5006,14 @@ private boolean isUpstreamSourceDir(File dirAdded) {
     }
 
     // prints error to users when Maven thread classpath error is present
-    private void printMavenClasspathErrMsg() {
-        error("A classpath error with Maven has been detected. Tests will not run in dev mode on this project. Run tests outside of dev mode with `mvn verify` or `mvn failsafe:integration-test`.");
+    private void printMavenClasspathErrMsg(boolean startup) {
+        if (startup) {
+            String message1 = "A classpath error with Maven has been detected. Tests will not run in dev mode on this project.";
+            String message2 = "Run tests outside of dev mode with `mvn verify` or `mvn failsafe:integration-test`.";
+            error(startup ? formatAttentionMessage(message1) : message1);
+            error(startup ? formatAttentionMessage(message2) : message2);
```

Comment:
In this code block within the `if` condition, `startup` is already `true` so only the `formatAttentionMessage()` part is being used here.